flink-contrib/flink-storm-compatibility/flink-storm-compatibility-core/src/main/java/org/apache/flink/stormcompatibility/wrappers/StormBoltWrapper.java

```diff
@@ -205,7 +211,35 @@ public void open(final Configuration parameters) throws Exception {
 					this.numberOfAttributes, flinkCollector));
 		}
 
-		this.bolt.prepare(null, topologyContext, stormCollector);
+		Map stormConf = new HashMap();
```

Comment:
Seems to be the same code as in `AbstractStormSpoutWrapper.open(...)`. Can you unify it, ie, add a new static method to `StormWrapperSetupHelper` that does it, and just call it here.